api/controllers/TelevisionController.js

```diff
@@ -0,0 +1,328 @@
+/** 
+  * Gladys Project
+  * http://gladysproject.com
+  * Software under licence Creative Commons 3.0 France 
+  * http://creativecommons.org/licenses/by-nc-sa/3.0/fr/
+  * You may not use this software for commercial purposes.
+  * @author :: Pierre-Gilles Leymarie
```

Comment:
Can you remove this license banner that is no longer valid ? :)

Reply:
@piznel fix this on commit a2c78df04f3ae0157137d396bcc132b63a0fbcd3